src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalPrice = 0;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const DAILY_CAR_RENT = 40;
+
+  for (let i = 0; i < days; i++) {
+    totalPrice += DAILY_CAR_RENT;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    totalPrice -= SHORT_TERM_DISCOUNT;
```

Comment:
don't need to change totalPrice in every case, you can just return new data
```suggestion
    return totalPrice - SHORT_TERM_DISCOUNT;
```